run/GCHP/HEMCO_Config.rc.templates/HEMCO_Config.rc.fullchem

```diff
@@ -3853,7 +4051,7 @@ ${RUNDIR_TES_CLIM_N2O}
 #==============================================================================
 # --- Annual scale factors ---
 #==============================================================================
-(((HTAP.or.XIAO_C3H8
+(((HTAPv3.or.HTAPv3_SHIP.or.XIAO_C3H8
```

Comment:
See comment in GCClassic version of this file.

Reply:
Now fixed in 3f69125
https://github.com/geoschem/geos-chem/blob/3f69125dbdf6ae180be14d943941683f4c7b9185/run/GCHP/HEMCO_Config.rc.templates/HEMCO_Config.rc.fullchem#L4051-L4067

Reply:
@yantosca Could you also remove the obsolete scale factors from this section? That is, only scale IDs 6 and 7 should remain since those are the only annual scale factors used by XIAO_C3H8.

Reply:
Now removed in https://github.com/geoschem/geos-chem/commit/dd89da81003b7d9ebe96af4de90ecaf7620d290e